01-shapefile-attributes.Rmd

```diff
@@ -75,182 +79,292 @@ CSV to Shapefile in R</a>
-More about the NEON Harvard Forest field site</a>. These data are proxy data for
-what will be available for 30 years from the NEON flux tower [from the NEON data portal](http://data.neoninc.org/ "NEON data").
-{: .notice}
- 
-##Notes about R Libraries
+###Additional Resources
 
-To work with vector data in R, we can use the `rgdal` library. We will load the 
-`raster` library to work with rasters. The `raster` library also allows us to 
-explore metadata using similar commands with both rasters and vectors.
-
-##Setting up the workspace
-If you completed Lesson 00  {{site.baseurl}}R/open-shapefiles-in-R/  you can skip this code chunk.
-```{r load-libraries-data }
+</div>
 
-library(rgdal)
-library (raster)   #notice sp package should always load with raster. 
-#in case the libraries and data are not still loaded
+#Shapefile Metadata & Attributes
+When we imported a shapefile layer into `R`, the `readOGR()` function 
+automatically stores informatation about and associated with the data. The
+geospatial *metadata* that contains information on the data set and
+*attributes* that contain non-geographic information about the items in the data
+set. 
+
+##Load the Data
+To work with vector data in `R`, we can use the `rgdal` library. The `raster` 
+package also allows us to explore metadata using similar commands for both
+rasters and vectors. We will shapefiles with data for a field site boundary,
+roads within the field site, and the tower location.
+If you completed the
+[Vector Data in R - Open and Plot Shapefiles]({{site.baseurl}}R/open-shapefiles-in-R/ ) 
+lesson, you can skip this code chunk.
+
+```{r load-packages-data }
+#load packages
+library(rgdal)  #for vector work; sp package should always load with rgdal. 
+library (raster)   #for metadata/attributes- vectors or rasters
+
+#set working directory to data folder
+#setwd("pathToDirHere")
 
 #Import a polygon shapefile 
-aoiBoundary <- readOGR("boundaryFiles/HARV/", "HarClip_UTMZ18")
+aoiBoundary_HARV <- readOGR("boundaryFiles/HARV/", "HarClip_UTMZ18")
 
 #Import a line shapefile
-lines <- readOGR( "boundaryFiles/HARV/",layer = "HARV_roads")
+lines_HARV <- readOGR( "boundaryFiles/HARV/", "HARV_roads")
 
 #Import a point shapefile 
-point <- readOGR("boundaryFiles/HARV/", "HARVtower_UTM18N")
+point_HARV <- readOGR("boundaryFiles/HARV/", "HARVtower_UTM18N")
 
 ```
 
-##Querying shapefile metadata 
-Recall from the previous lesson that shapefile metadata can include a `class`, 
-a `features` count, an `extent`, and a `coordinate reference system` (crs). 
-
-Metadata can be extracted individually using the `class()`, `length()`, 
-`extent()`, `crs()`, `R` commands. 
+#Query Shapefile Metadata 
+Shapefile metadata can include object `class`, a features count (`length`), the
+object `extent`, and the object's coordinate reference system (`CRS`). To query
+the shapefile metadata the code is s
 
 ```{r view-shapefile-metadata }
```

Comment:
Rather than do each step 3 times. why don't we show them with a point. Then add a chalenge that asks for the CRS, class and length, extent of the lines and polygons layers. that will be better.